pegasus/sites.v3/hourofcode.com/public/events/2013/index.haml

```diff
@@ -3,38 +3,25 @@ title: <%= hoc_s(:events_all_title) %>
 layout: wide
 nav: events_nav
 ---
--SOLR = Solr::Server.new(host:CDO.solr_server)
+:ruby
+  require pegasus_dir 'helper_modules/forms'
+  hoc_year = 2013
+  kind = 'CSEdWeekEvent2013'
```

Comment:
nit: should these be constants?

Same with similar static values in other forms below.